.circleci/config.yml

```diff
@@ -0,0 +1,22 @@
+# GCB-Dockerfiles CircleCI 2.0 configuration file
+version: 2
+jobs:
+  build:
+    docker:
+      - image: docker:17.05.0-ce-git
```

Comment:
Is this particular image necessary? Does not using `setup_remote_docker` with one of the standard circleci images suffice?

Reply:
I looked at the standard CircleCI images. Most are language-specific. For a python project you'd use a python image. Since the language of this repo is Docker, it made since to use a Docker image. I followed the recommendation from https://circleci.com/blog/how-to-build-a-docker-image-on-circleci-2-0/.

> What we want is a Docker image that installs Docker and has Git. These requirements are satisfied by using `docker:17.05.0-ce-git`, which is an [official Docker image](https://hub.docker.com/_/docker/).
> 
> When an image has the suffix “-git”, it means Git is pre-installed. By using this image, you ensure you’re always using the latest Docker client.

I did [try](https://circleci.com/gh/dleehr/GCB-Dockerfiles/8) `buildpack-deps:trusty` when I noticed that docker/alpine image didn't have bash. But the buildpack-deps image didn't have docker installed. The `-dind` variants probably do, but I thought it made more sense just to use the docker one.